src/Frontend/Core/Engine/Header.php

```diff
@@ -132,7 +132,11 @@ public function addCSS($file, $minify = true, $addTimestamp = null)
         $file = (string) $file;
         $minify = (bool) $minify;
         $addTimestamp = (bool) $addTimestamp;
-        $file = Theme::getPath($file);
+
+        // get file path
+        if (mb_substr($file, 0, 4) != 'http') {
```

Comment:
I ran extensive tests and found there is no performance difference between the two methods. Also, looking at the semantics, there is no major difference, it's a potato, potato story. Do you have any arguments as to why yours is better, @carakas ?